PUBLICATIONS.md

```diff
@@ -0,0 +1,66 @@
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/security-whitepaper/v2/cloud-native-security-whitepaper.md)
+- [PDF](https://www.cncf.io/wp-content/uploads/2022/06/CNCF_cloud-native-security-whitepaper-May2022-v2.pdf)
+- [Audio](https://soundcloud.com/user-769472014/sets/cncf-tag-security-cloud-native-security-whitepaper-version-v1)
+
+## Supply Chain Security
+
+### Software Supply Chain Best Practices
+
+The Software Supply Chain Security Paper is a TAG Security effort to ensure the cloud native community has access to information about building, distributing, deploying, and running secure software supply chains.
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/supply-chain-security/supply-chain-security-paper/sscsp.md)
+- [PDF](https://github.com/cncf/tag-security/raw/main/supply-chain-security/supply-chain-security-paper/CNCF_SSCP_v1.pdf)
+
+### Evaluating your supply chain security
+
+A framework for supply chain evaluation
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/supply-chain-security/supply-chain-security-paper/secure-supply-chain-assessment.md)
+
+### Secure Software Factory
+
+A reference architecture for securing the software supply chain
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/supply-chain-security/secure-software-factory/secure-software-factory.md)
+- [PDF](https://github.com/cncf/tag-security/raw/main/supply-chain-security/secure-software-factory/Secure_Software_Factory_Whitepaper.pdf)
+
+## Cloud Native Security Lexicon
+
+Standardization of terminologies specific to Cloud Native Security
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/security-lexicon/cloud-native-security-lexicon.md)
+
+## Use Cases & Personas
+
+List of use cases to enable secure access, policy control and safety for users of cloud native technology
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/usecase-personas/README.md)
+
+## Policy
+
+### Formal Verification for Policy Configurations
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/policy/overview-policy-formal-verification.md)
+
+### Handling build-time dependency vulnerabilities
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/policy/overview-policy-build-time-dependency-vulns.md)
+
+## Secure Defaults: Cloud Native 8
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/security-whitepaper/secure-defaults-cloud-native-8.md)
+
+## Cloud Native Security Controls Catalog
+
+Mapping of Cloud Native Security Whitepaper and Software Supply Chain Best Practices Paper to NIST SP800-53r5
+
+- [Markdown](https://github.com/cncf/tag-security/blob/main/cloud-native-controls/phase-one-announcement.md)
+- [Spreadsheet](https://docs.google.com/spreadsheets/d/1GUohOTlLw9FKUQ3O23X7ypvJLXN-B3veJGe6YE6JYfU/edit?usp=sharing)
```

Comment:
Thanks @szh for this PR. Much appreciated. 
Please incorporate the suggestions in this comment to reference some more efforts from the community 
https://github.com/cncf/tag-security/issues/1026#issuecomment-1382619123